File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/queryvb/vb/linqexamples32.vb
' <snippetlinqexamples32>


 Using svcContext As New ServiceContext(_serviceProxy)
  Dim list_retrieve1 = From c In svcContext.ContactSet _
                       Where c.ContactId.Value.Equals(_contactId1) _
                       Select New With
                              {Key .StatusReason =
                                  c.FormattedValues("statuscode")}
  For Each c In list_retrieve1
   Console.WriteLine("Status: " &amp; c.StatusReason)
  Next c
 End Using

' </snippetlinqexamples32>

File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/query/cs/linqexamples32.cs
// <snippetlinqexamples32>


 using (ServiceContext svcContext = new ServiceContext(_serviceProxy))
 {
  var list_retrieve1 = from c in svcContext.ContactSet
                       where c.ContactId == _contactId1
                       select new { StatusReason = c.FormattedValues["statuscode"] };
  foreach (var c in list_retrieve1)
  {
   System.Console.WriteLine("Status: " + c.StatusReason);
  }
 }

// </snippetlinqexamples32>